.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 sudo: false
 cache: bundler
-script: 'script/ci && bundle exec rubocop'
+script: 'bundle exec rake spec:coverage --trace && bundle exec rubocop'
```

Comment:
This should stay `script/ci`. It guarantees isolation for `multi_json` specs et al.